app/views/my_facilities/missed_visits.html.erb

```diff
@@ -0,0 +1,130 @@
+<div class="row">
+  <div class="col-lg-12 pr-lg-0">
+    <div class="card card-responsive">
+      <div class="row">
+        <div class="col-lg-9 pr-5 pr-md-0">
+          <h3>Missed visit</h3>
+          <p>Patients who had no BP recorded during the quarter as a pecentage of all-time registered patients (patients
+            with no visit in > 1 year are excluded).</p>
+        </div>
+
+        <div class="mt-1 mb-3 col-lg-3">
+          <div class="dropdown show">
+            <a class="btn btn-sm btn-outline btn-secondary dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style="min-width: 12em;">
+              <%= PeriodSelection::PERIODS[:missed_visits][@selected_period] %>
+            </a>
+
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+              <% PeriodSelection::PERIODS[:missed_visits].each do |period, period_name| %>
+                <button type="submit" name="period" value=<%= period %> class="dropdown-item" form="query-filters"><%= period_name %></button>
+              <% end %>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <table class="mt-3 mt-lg-4 table table-compact table-responsive-md table-hover" id="analytics-table">
+        <colgroup>
+          <col>
+          <% @display_periods.each do %>
+            <col class="table-divider">
+            <% unless @selected_period == :day %>
+              <col>
+            <% end %>
+          <% end %>
+          <col class="table-divider">
+          <col class="table-divider">
+          <col class="mobile">
+        </colgroup>
+        <thead>
+        <tr>
+          <th></th>
+          <% case @selected_period %>
+          <% when :quarter %>
+            <% @display_periods.reverse.each do |(year, quarter)| %>
+              <th colspan="2"><%= "Q#{quarter}-#{year}" %></th>
+            <% end %>
+          <% when :month %>
+            <% @display_periods.reverse.each do |(year, month)| %>
+              <th colspan="2"><%= month_short_name_and_year(local_month_start(year, month)) %></th>
+            <% end %>
+          <% end %>
+          <th>Calls made <sup>1</sup></th>
+          <th>ALL-TIME</th>
```

Comment:
I recommend upcasing this through CSS, what do you think?

Reply:
Hmm, not sure that's necessary, given that its just a static string. Does upcasing this through CSS do something else I'm missing?

Reply:
Just separates the content from the "presentation". I think it's a good practice to follow.

Reply:
Turns out this was already being uppercased by some table header css, just changed it to `All-time` in the html.